client/src/Reviews.jsx

```diff
@@ -12,16 +30,18 @@ export default class Reviews extends React.Component {
     };
   }
 
+  /* -------------------------  RENDER ------------------------- */
+
   render() {
     return (
-      <div>
+      <ContainerDiv>
         <div>
-          <h2>Customer Reviews</h2>
+          <Title>Customer Reviews</Title>
         </div>
-        <AverageRating average={'/* TODO */'} ratings={'/* TODO */'} />
+        <AverageRating average={4.3} ratings={399} />
```

Comment:
These numbers will be provided to the component dynamically via a fetch/axios call to the server.